scripts/drone/events/release.star

```diff
@@ -203,6 +205,9 @@ def publish_packages_pipeline():
         compile_build_cmd(),
         publish_linux_packages_step(package_manager = "deb"),
         publish_linux_packages_step(package_manager = "rpm"),
+        # JEV: is this hardcoded arg going to be an issue?
+        verify_linux_DEB_packages_step(depends_on = []),
+        verify_linux_RPM_packages_step(depends_on = []),
```

Comment:
These need to depend on the publish steps (since we have to publish the linux packages before we can test them!)

Reply:
I inserted those as default `depends_on` args in the starlark functions themselves in lib.star. Is it better to have them here instead?
```
def verify_linux_DEB_packages_step(depends_on = ["publish-linux-packages-deb"]):
    return {
...
```

Reply:
you should define them in the pipeline here. It makes it clearer.

BTW like python if you set it like this: `verify_linux_DEB_packages_step(depends_on = [])`, that means `depends_on` will be empty (and not the default). You'd leave it default by not specifying anything. `verify_linux_DEB_packages_step()`